docs/data/sources/google-tag-manager-client.md

```diff
@@ -39,7 +39,7 @@ Note: We do *not* recommend installing amplitude through a Custom HTML tag.
  
 #### API Key
 
-Copy your amplitude API Key in the API Key field. For EU residency, please make sure your API Key is under analytics.eu.amplitude.com.
+Copy your amplitude project API Key in the API Key field. For EU residency, your project API Key is under analytics.eu.amplitude.com. Each project has different API Key, please make sure you are copy the API Key from the right project. Go to **Settings -> Projects -> click the right project name from the list** to find your project API Key. 
```

Comment:
Maybe make `analytics.eu.amplitude.com` a link? Or call out using "code style" with tikmarks?

Reply:
How about adding the link https://www.docs.developers.amplitude.com/analytics/find-api-credentials/?